resources/scss/components/_external-icon.scss

```diff
@@ -0,0 +1,52 @@
+.external-icon {
+    content: '(external link)';
+    display: inline-block;
+    background-color: currentColor;
+    mask-image: url('/_resources/images/external-link.svg');
+    mask-repeat: no-repeat;
+    mask-position: center;
+    mask-size: contain;
+    line-height: 0;
+    width: 0.7em;
+    height: 0.7em;
+    position: relative;
+    bottom: 0.29em;
+    left: 0.25em;
+}
+
+// Strictly inline-block anchors
+.main-menu .external-link a,
+.content a.external-link[class*="button"],
+.under-menu a.external-link[class*="button"] div:first-of-type {
+    &::before {
+        @apply hidden;
+    }
+
+    &::after {
+        @extend .external-icon;
+    }
+}
+
+// Strictly inline anchors
+.content a.external-link {
```

Comment:
For consistency, let's continue to only use the class `.external`:
https://base.wayne.edu/styleguide/menu/external

Reply:
Please explain how I need to be consistent with something that is a new idea? I suggest the specificity of external-link, so if you mean changing the styleguide url to match I can do that.